### DISCUSSION TITLE
AP do not show after flash

### DISCUSSION THREAD

**Discussion by lennart-bystrom** [arendst/Tasmota#22239] [Q&A] [answered]
Hi, newbe so probbaly user error :-(  
I have a Sonoff Zigbeebridge Pro that I have installed tasmota-zbbridge.bin on with the online tool https://tasmota.github.io/install/.
I get the message that the install is successfull, after that I have disconected the seriel/usb device and conected the usb port on the sonoffbridge to a usb that power the sonoff. After this I have understood that a wifi named tasmota.... should show, but no such wifi is availible. Dont know if it has any bearing but I tried first to flash with ESP-Flasher and tasmotizer-1.2 but non of them succeded, it connects but the fails. any help much appriciated!

**Comment by sfromis** [comment, marked as answer]
If you flashed `tasmota-zbbridge.bin` on a Sonoff Zigbee Bridge "pro", that's your issue, as you need a binary for ESP32 instead of one for ESP8266. I'd suggest to select "Tasmota32 Sonoff-ZigbeeBridgePro" in the Tasmota Web Intaller, alternatively pick `tasmota32-zbbrdgpro.factory.bin` yourself. 

Tasmotizer does not work for ESP32, and ESP-Flasher needs you to know what you're doing, which does not appear to be the case :wink:

**Reply by lennart-bystrom** [reply]
:-) no I do not know what I'm doing but persitant ;-) and with some help from freandly people like your self I'm now one step further, the wifi is up and the module is conected to my own IOT vlan :-) many thx!!
